sdk/containerregistry/container-registry/samples/typescript/src/containerRepositoryClient.ts

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary Demonstrates the use of a ContainerRegistryClient.
+ */
+
+import { ContainerRepositoryClient } from "@azure/container-registry";
+import { DefaultAzureCredential } from "@azure/identity";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+export async function main() {
+  const endpoint = process.env.ENDPOINT ?? "<endpoint>";
+  const repository = process.env.REPOSITORY_NAME ?? "<repository name>";
+
+  const client = new ContainerRepositoryClient(endpoint, repository, new DefaultAzureCredential());
+  await listTags(client);
+  await listArtifacts(client);
+  await getProperties(client);
+  const digest = "sha256:4661fb57f7890b9145907a1fe2555091d333ff3d28db86c3bb906f6a2be93c87";
```

Comment:
Where does this digest come from? Can we use a digest we get from `listArtifacts`  instead so we don't need to hardcode `digest`?